Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/azuredeploy.json

```diff
@@ -0,0 +1,2862 @@
+            "5. Click Save",
+            "6. Repeat steps for other connections",
+            "**b. Assign Role to add comment in incident**",
+            "After authorizing each connection, assign role to this playbook.",
+            "1. Go to Log Analytics Workspace → <your workspace> → Access Control → Add",
+            "2.Add role assignment",
+            "3.Assignment type: Job function roles",
+            "4.Role: Microsoft Sentinel Contributor",
+            "5.Members: select managed identity for assigned access to and add your logic app as member as shown in screenshot below",
+            "6.Click on review+assign",
+            "**c. Add Access policy in Keyvault**",
+            "Add access policy for playbook's managed identity to read, write secrets of keyvault.",
+            "1. Go to logic app → <your logic app> → identity → System assigned Managed identity and copied it.",
+            "2. Go to keyvaults → <your keyvault> → Access policies → create.",
+            "3. Select all keys & secrets permissions. Click next.",
+            "4. In principal section, search by copied identity object ID. Click next.",
+            "5. Click review + create.",
+            "**d. Configurations in Sentinel**",
+            "1. ASimNetworkSession, ASimDns, ASimWebSession & imFileEVent ASIM Parsers must be available in workspace to use an analytic rules which will trigger this playbook.",
+            "2. In Microsoft sentinel, analytical rules should be configured to trigger an incident based on ASimNetworkSession, ASimDns, ASimWebSession & imFileEvent.",
+            "3. Configure the automation rule to trigger the playbook."
+        ],
+        "entities": [
+            "IP", "Host", "URL", "FileHash"
+        ],
+        "tags": [
+            "Dataminr", "Pulse", "alerting", "enrichment"
+        ],
+        "support": {
+            "tier": "community",
+            "armtemplate": "Generated from https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator"
+        },
+        "author": {
+            "name": "Dataminr"
+        }
+    },
+    "parameters":  {
+        "PlaybookName":  {
+            "defaultValue":  "DataminrPulseAlertEnrichment",
+            "type":  "string"
+        },
+        "KeyVaultName":  {
+            "type":  "string",
+            "minLength": 1
+        },
+        "tenantId": {
+            "type": "string",
+            "minLength": 1
+        },
+        "BaseURL": {
+            "type": "string",
+            "minLength": 1,
+            "metadata": {
+                "description": "Enter baseurl for your Dataminr account.(e.g. https://gateway.dataminr.com/)"
+            }
+        }
+    },
+    "variables":  {
+        "MicrosoftSentinelConnectionName":  "[concat('MicrosoftSentinel-', parameters('PlaybookName'))]",
+        "KeyvaultConnectionName":  "[concat('Keyvault-', parameters('PlaybookName'))]",
+        "Keyvault-audience": "vault.azure.net"
```

Comment:
Hello @jayeshprajapaticrest value set for variable "Keyvault-audience" is "vault.azure.net". 
That is where the arm ttk is failing.
Does that variable should refer to the parameter "BaseURL" ???

Reply:
@v-prasadboke No. It does not refer to the parameter BaseUrl as this Url is for APIs used in the playbook.